src/data_processing/wiki_to_netflix.py

```diff
@@ -0,0 +1,133 @@
+import requests
+import csv
+import os
+import math
+
+base_dir = os.path.join(os.path.dirname(__file__), '../data/')
+user_agent = 'Noisebridge MovieBot 0.0.1/Audiodude <audiodude@gmail.com>'
+
+# Reading netflix text file
+def read_netflix_txt(txt_file, test):
+    netflix_list = []
+    num_rows = 100 if test else math.inf
```

Comment:
It's a bit messy to se num_rows to infinity. It would be better to just set it to `None` if `test == True` and check for that in the condition below.

Reply:
That is fair tbh. I didn't realize that you could compare integers to `None` in python, and that it apparently will work as intended. I agree the inline if/ else statement is a bit messy/ lazy as well. This was kinda just like minor tech debt that was there because it technically worked, but yeah should be fixed now.

Reply:
See comment below for a suggested way to write the if, if num_rows is `None`